test/dispatcher.js

```diff
@@ -9,11 +21,10 @@ suite('Event Generation and Dispatching', function() {
     document.body.appendChild(container);
   });
 
-  teardown(function() {
+  after(function() {
     document.body.removeChild(container);
   });
 
-  var pepde = PointerEventsPolyfill.dispatcher.eventSources;
```

Comment:
thanks, I added the line back, and modified it to var pepde = pep.dispatcher.eventSources;